fe/fe-core/src/main/java/com/starrocks/sql/analyzer/FunctionAnalyzer.java

```diff
@@ -108,12 +119,12 @@ private static void analyzeBuiltinAggFunction(FunctionCallExpr functionCallExpr)
             }
         }
 
-        if (fnName.getFunction().equalsIgnoreCase("dense_rank")
```

Comment:
has `fnName` already convert to low case?

Reply:
Yes. All function names should be lower case.